src/js/utils/JestUtil.js

```diff
@@ -88,6 +142,42 @@ const JestUtil = {
   },
 
   /**
+   * Generates a callback function to a reduce() call that will
+   * find and return the textContent of DOM element(s) matching
+   * the given selector
+   *
+   * @example <caption>How to use reduceTextContentOfSelector</caption>
+   * const TestUtils = require('react-addons-test-utils');
+   *
+   * var componentInstance = TestUtils.renderIntoDocument(
+   *     <MyTableComponent />
+   *   );
+   *
+   * var names = TestUtils.scryRenderedDOMComponentsWithClass(
+   *       componentInstance, 'table-column-name')
+   *       .reduce(reduceTextContentOfSelector(
+   *          '.collapsing-string-full-string'),
+   *          []
+   *       );
+   *
+   * @param {string} selector - The CSS selector to match children
+   * @returns {function} Returns a callback function to be passed on .reduce()
+   */
+  reduceTextContentOfSelector(selector) {
+    return function (strings, element) {
+      let stringContents = element.querySelectorAll(selector);
```

Comment:
No, because the reduce function will be applied on array of elements, each one of them might contain more than one possible match:

```javascript
var elements = [ DOMElement, DOMElement ];
var texts = elements.reduce(reduceTextContentOfSelector("span"), []);
// texts can contain more than 2 elements, since each one of the parent
// dom elements can have more than 1 <span> inside them.
```